scripts/systemtap/kem/kem_run.sh

```diff
@@ -70,14 +70,14 @@ sleep 2
 
 for i in $(seq 0 $(($(nproc)-1)))
 do
-    rm -f /dev/kemd$i
+    rm -f /dev/kemd"$i"
```

Comment:
This looks strange.
Do you prefer:
`rm -f "/dev/kemd$i"`
?

Reply:
done